src/StokesDrifts.jl

```diff
@@ -73,7 +73,7 @@ UniformStokesDrift{Nothing, typeof(uniform_stokes_shear), typeof(Oceananigans.St
 ```
 
 Exponentially-decaying Stokes drift corresponding to a surface Stokes drift of
-`uˢ₀ = 0.005` and decay scale `h = 20 m`, using parameters:
+`uˢ = 0.005 m s⁻¹` and decay scale `h = 20 m`, using parameters:
```

Comment:
i don't think we should put this in, because the units depend on the grid (and other parameters), none of which are specified here

Reply:
OK
I only did that because it was like that for `h = 20 m`. then let's remove it from there as well?

Reply:
ok done